system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,13 +321,54 @@
 }
 
 .topsite-form {
+  .form-input-container {
+    display: flex;
+    max-width: 492px;
+    margin: 0 auto;
+    padding: 32px;
+
+    .top-site-outer {
+      width: 96px;
+      padding: 0;
+      margin: 64px 0 0;
+      margin-inline-start: 32px;
```

Comment:
Thought I had a comment here but seems like it didn't make it into the review…? This particular positioning seems odd. Is it expected to line up with the top of the first input box? What happens when there's more lines or more inputs?

![screen shot 2018-02-12 at 3 20 25 pm](https://user-images.githubusercontent.com/438537/36126603-1803e65a-100e-11e8-835b-84c23000cf1c.png)

Reply:
To fix this I added a media query so that on a narrow design the preview moves below the inputs.
This seems good enough for our current fields. Even if we add more fields with longer labels (Custom Screenshot URL might be one culprit) it won't break alignment since only the first input matters.
Design feedback:
> does this look ok for a narrow view?
> Yeah that’s fine

![screen shot 2018-02-13 at 17 11 31](https://user-images.githubusercontent.com/810040/36160247-f81362e8-10e0-11e8-967d-8081401a1bd6.png)